packages/sources/deribit/src/adapter.ts

```diff
@@ -1,23 +1,23 @@
 import { Config, ExecuteFactory, ExecuteWithConfig } from '@chainlink/types'
 import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { makeConfig } from './config'
 
 const customParams = {
   currency: ['base', 'from', 'coin', 'symbol'],
 }
 
-export const execute: ExecuteWithConfig<Config> = async (input, config) => {
-  const validator = new Validator(input, customParams)
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
   const currency = validator.validated.data.currency
 
-  const url = 'https://www.deribit.com/api/v2/public/get_historical_volatility'
```

Comment:
`BASE_URL` is now `'https://api.currencylayer.com'` and `URL` is now  `get_historical_volatility`.
The middle of `api/v2/public/` is getting lost, It should be added to the `BASE_URL`.